app/views/organisations/_contacts.html.erb

```diff
@@ -21,7 +21,7 @@
         <div class="<%= contact[:post_addresses].any? ? "govuk-grid-column-one-half" : "govuk-grid-column-two-thirds" %>">
           <% if contact[:email_addresses].any? %>
             <div class="organisation__margin-bottom">
-              <p><%= t('organisations.contact.email') %></p>
+              <p lang="en"><%= t('organisations.contact.email', locale: :en) %></p>
```

Comment:
Picky and not a blocker - could we change the `lang` attribute to only be displayed if the outer element is not in English?